src/app/_components/home/activity/activity-home.css

```
@@ -189,58 +189,167 @@
 /* --------------------------------------------------------------------------
    WEEK-IN-REVIEW
    -------------------------------------------------------------------------- */
+/* The week card gets a distinct surface from the shared .wsa-card chrome:
+   a darker base with a faint top-left brand glow, slightly larger radius and
+   padding, to read as the dashboard's headline panel. */
```

Stylelint (17.12.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)